app/models.py

```diff
@@ -642,7 +642,7 @@ def __init__(self, **kwargs):
     service = db.relationship('Service', backref='service_callback')
     url = db.Column(db.String(), nullable=False)
     callback_type = db.Column(db.String(), db.ForeignKey('service_callback_type.name'), nullable=True)
-    _bearer_token = db.Column("bearer_token", db.String(), nullable=False)
+    _bearer_token = db.Column("bearer_token", db.String(), nullable=True)
```

Comment:
Is there protection in place to capture null (an empty string) where this code is called/used?

Reply:
@k-macmillan There is a code to capture null at line 657
```
    @property
    def bearer_token(self):
        if self._bearer_token:
            return encryption.decrypt(self._bearer_token)
        return None

    @bearer_token.setter
    def bearer_token(self, bearer_token):
        if bearer_token:
            self._bearer_token = encryption.encrypt(str(bearer_token))
```

Reply:
```
op.alter_column('service_callback', 'bearer_token',
               existing_type=sa.VARCHAR(),
               nullable=False)
    op.alter_column('service_callback_history', 'bearer_token',
               existing_type=sa.VARCHAR(),
               nullable=False)
```
I didn't make this change. @kalbfled pushed this. But as I am checking, above unexpected output has been addressed by this change.

Reply:
I created the PR from Maham's branch after she left the team, and I have not previously inspected any of the changes for correctness.

I have the same question as before: How did it get like this in the first place?  If `bearer_token` should be non-nullable but isn't, that "alter_column" operation is appropriate.  How has it not been applied already?  Or maybe it has in the database, but Alembic doesn't recognize it.  We should alter the model, if correct, to eliminate migration output.

`nullable=True` is the default.  You could just delete `nullable=False`.